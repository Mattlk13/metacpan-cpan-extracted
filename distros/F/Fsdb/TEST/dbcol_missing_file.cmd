prog='dbcol'
args='--input TEST/nonexistant_file account'
cmp='diff -c -b '
in=/dev/null
expected_exit_code=fail
