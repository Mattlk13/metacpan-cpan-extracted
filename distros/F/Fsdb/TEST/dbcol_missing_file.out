dbcol: cannot open input: cannot open TEST/nonexistant_file
